src/components/layout/Navbar.tsx

```diff
@@ -42,29 +61,50 @@ const navItems: NavItem[] = [
     label: 'Verification',
     url: '/verification',
   },
+  {
+    label: 'Settings',
+    url: '/settings',
+  },
   {
     label: 'SearchSECO',
     pages: [
-      { label: 'Query', url: '/query' },
-      { label: 'Mining', url: '/mining' },
+      {
+        label: 'Query',
+        url: '/query',
+        icon: HiOutlineDocumentMagnifyingGlass,
+        description: `Query the SearchSECO database using your ${TOKENS.secoin.symbol} tokens.}`,
+      },
+      {
+        label: 'Mining',
+        url: '/mining',
+        icon: HiOutlineTerminal,
+        description: `Claim your mining rewards in ${TOKENS.secoin.symbol} or ${TOKENS.rep.symbol}`,
+      },
+    ],
+    alternativeLinks: [
+      { label: 'Website', url: '#', icon: HiOutlineGlobeAlt },
+      { label: 'Discord', url: '#', icon: FaDiscord },
     ],
-  },
-  {
-    label: 'Settings',
-    url: '/settings',
   },
 ];
 
-const Navitempage = ({ item }: { item: NavItemPage }) => {
+const Navitempage = ({
```

Comment:
What happened to naming convention for components??
Same for the other components in this file

Reply:
Dat was @BenjaminMeijs whoops, anders had de naam conflict met de types